src/schema/v2/user.ts

```diff
@@ -43,6 +43,24 @@ export const UserAdminNoteType = new GraphQLObjectType<any, ResolverContext>({
       type: new GraphQLNonNull(GraphQLString),
     },
     createdAt: date(({ created_at }) => created_at),
+    adminName: {
+      description: "The name of the admin user who created the note",
+      type: GraphQLString,
+      resolve: async ({ created_by }, {}, { userByIDLoader }) => {
+        if (!userByIDLoader) {
+          throw new Error(
+            "Loader not found. You must supply an X-Access-Token header."
+          )
+        }
+
+        if (!created_by) {
+          return null // This is a legacy note, associated user is not available
```

Comment:
As mentioned in the PR description, all [ResourceNote](https://github.com/artsy/gravity/blob/main/app/models/domain/resource_note.rb)(s) up to now have not set `created_by` field and so MP will return `adminName: null` for all of the 'legacy' notes.

Reply:
Good call 👍 !